openedx/core/djangoapps/catalog/utils.py

```diff
@@ -45,8 +56,15 @@ def get_programs(uuid=None):
             logger.warning(missing_details_msg_tpl.format(uuid=uuid))
 
         return program
-
-    uuids = cache.get(PROGRAM_UUIDS_CACHE_KEY, [])
+    if waffle.switch_is_active("get-multitenant-programs"):
+        logger.info('Fetching programs from cache for site {site_name}, key = {cache_key}'.format(
+            site_name=get_current_site().domain,
+            cache_key=SITE_PROGRAM_UUIDS_CACHE_KEY.format(site_name=get_current_site().domain)
+        ))
+        uuids = cache.get(SITE_PROGRAM_UUIDS_CACHE_KEY.format(site_name=get_current_site().domain), [])
```

Comment:
Let's get this straight: are sites referred to by their name or by their domain? Using the domain seems fine, but if you're going to do that, please make the format variable consistent (i.e., `domain`).